src/main/java/lotto/model/lotto/winningResult/rank/Rank.java

```diff
@@ -0,0 +1,57 @@
+package lotto.model.lotto.winningResult.rank;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public enum Rank {
+    FIRST(1, 6, false, 2000000000),
+    SECOND(2, 5, true, 30000000),
+    THIRD(3, 5, false, 1500000),
+    FOURTH(4, 4, false, 50000),
+    FIFTH(5, 3, false, 5000),
+    FAIL(6, -1, false, 0);
```

Comment:
```suggestion
    FIRST(1, 6, false, 2_000_000_000),
    SECOND(2, 5, true, 30_000_000),
    THIRD(3, 5, false, 1_500_000),
    FOURTH(4, 4, false, 50_000),
    FIFTH(5, 3, false, 5_000),
    FAIL(6, -1, false, 0);
```
와 같이 작성한다면, 큰 숫자를 표현할 때 더 가독성이 좋을 수 있을 것 같네요~!

추가적으로` private final int rank;` 해당 필드는 어떤 역할을 하는지 알 수 있을까요? 단순 등수 표현일까요?

Reply:
다른 분들도 이렇게 하셨더라구요 ㅎㅎ 처음알게된 사실이에요 감사합니다

rank를 지정한 이유는 등수표현이기도 하면서, 전체 결과를 정렬해서 출력할때 사용하는 도구예요!
출력예시보면 5등부터 1등 순으로 출력하더라구요!
물론 지금 상태에서 Rank.values()의 역순으로 출력하면 되긴 하는데, 저는 enum이 어떤 순서이건간에 한번 내림차순 정렬을 하고 싶어서 rank를 따로 지정했습니당

Reply:
@Chaeyoung714
원래 Enum 순서대로 0부터 시작하는 ordinal()이라는 값을 자동으로 가지게 되는것으로 알고 있긴 한데, enum 상수의 선언 순서가 바뀌면 값도 바뀌므로, 비즈니스 로직에서 사용하는 것은 권장되지 않는다고 하네요! 채영님 말처럼 명시적으로 `rank`라는 의미를 전달해주는 편이 더 나은 것 같네요!